org-ql.el

```diff
@@ -938,10 +950,20 @@ Arguments STRING, POS, FILL, and LEVEL are according to
               :action (lambda ()
                         (throw 'found t)))))))))
 
+(org-ql--defpred parent (querypredicatefn)
```

Comment:
Let's name the variable something simpler, like `predicate`.  :)

Reply:
Done